data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -13,6 +27,32 @@ wait_for_ha_api() {
     done
 }
 
+
+get_topology() {
+    output=$(oc --kubeconfig="$KUBECONFIG" get infrastructures cluster -o jsonpath='{.status.infrastructureTopology}' 2>&1 )
```

Comment:
I think we want `controlPlaneTopology` not `infrastructureTopology`

Reply:
Yeah that would be safer, since SNO with workers is a valid configuration